autopush/endpoint.py

```diff
@@ -254,6 +271,41 @@ def _db_error_handling(self, d):
         d.addErrback(self._response_err)
         return d
 
+    def _process_auth(self, result):
```

Comment:
Actually, if that was:
```python
if 'crypto-key' not in self.request.headers or 'authorization' not in self.request.headers:
    return result
```

Would wipe out both levels of if statements and indentation....